cmd/cluster-capi-operator/main.go

```diff
@@ -298,24 +298,24 @@ func setupReconcilers(mgr manager.Manager, infra *configv1.Infrastructure, platf
 		Platform:                    platform,
 		Infra:                       infra,
 	}).SetupWithManager(mgr); err != nil {
-		klog.Error(err, "unable to create controller", "controller", "CoreCluster")
+		klog.Error(err, "unable to create core cluster controller", "controller", "CoreCluster")
```

Comment:
I'm probably missing context, but each of the `klog.Error` calls here repeat the controller's name and `controller`, which seems redundant. Do we have a special handler installed? I don't see it in this file.

Reply:
It's a structured logging format here, so `controller` is adding a new key`, and the next string is the value. The logging through this function makes sense given the structured logging context